main.go

```diff
@@ -212,6 +212,16 @@ func FetchWorkerPools(context *workerpool.SubmitterContext) {
 			panic(err)
 		}
 		for _, workerPool := range workerPools.WorkerPools {
+  		// Strip out fields that change frequently
+  		workerPool.RequestedCapacity = 0
+  		workerPool.RequestedCount = 0
+  		workerPool.RunningCapacity = 0
+  		workerPool.RunningCount = 0
+  		workerPool.StoppedCapacity = 0
+  		workerPool.StoppedCount = 0
+  		workerPool.StoppingCapacity = 0
+  		workerPool.StoppingCount = 0
```

Comment:
Oh interesting. Nice fix, but too bad worker pools don't have a separate definition and status like tasks do (task status and task definition can be queried independently). We can't fix that here, but seems to be an oversight in the original Worker Manager design. :-(